certified-connectors/FlowForma V2/apiDefinition.swagger.json

```diff
@@ -0,0 +1,399 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "FlowForma V2",
+    "description": "FlowForma Business Process Management (BPM) tools for Office 365, enable users to build forms & workflow, automating processes without code.",
+    "contact": {
+      "name": "FlowForma Limited",
+      "email": "support@flowforma.com"
+    },
+    "version": "v1"
+  },
+  "host": "logic-apps.flowforma.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v1/getFlows": {
+      "get": {
+        "tags": [
+          "FormActions"
+        ],
+        "operationId": "GetFlows",
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "400": {
+            "description": "Bad request",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          },
+          "401": {
+            "description": "Unauthorized",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "summary": "Get flows",
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/v1/getSteps": {
+      "get": {
+        "tags": [
+          "FormActions"
+        ],
+        "operationId": "GetSteps",
+        "parameters": [
+          {
+            "in": "query",
+            "name": "flowId",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
```

Comment:
@FlowForma Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**